core/ir/instr_api.h

```diff
@@ -1549,10 +1549,17 @@ instr_zeroes_zmmh(instr_t *instr);
 
 DR_API
 /** Returns true if \p instr's opcode is #OP_xsave32, #OP_xsaveopt32, #OP_xsave64,
- * #OP_xsaveopt64, #OP_xsavec32 or #OP_xsavec64.
+ * #OP_xsaveopt64, #OP_xsavec32, #OP_xsavec64, #OP_xsaves32, or #OP_xsaves64.
  */
 bool
 instr_is_xsave(instr_t *instr);
+
+DR_API
+/** Returns true if \p instr's opcode is #OP_xrstor32, #OP_xrstor64, #OP_xrstors32,
```

Comment:
style violation: https://dynamorio.org/page_code_style.html#autotoc_md126 "For comments starting with /∗∗, leave the rest of the first line empty, unless the entire comment is a single line. "

Please fix the instr_is_xsave one too.

Reply:
Fixed. Also for instr_is_xsave above.